src/Editor.php

```
@@ -1887,4 +1925,417 @@ private function hasParameterNamed(array $params, string $name): bool
+        $method->name = new Node\Identifier($to, $method->name->getAttributes());
+        $renamed = 1;
+
+        foreach ((new NodeFinder())->find($owner, static fn (Node $n): bool => true) as $node) {
+            if (self::callsOwnMethod($node, $from)) {
+                $node->name = new Node\Identifier($to, $node->name->getAttributes());
+                ++$renamed;
+            }
+        }
+        $others = 0;
+
+        foreach ((new NodeFinder())->find($roots, static fn (Node $n): bool => true) as $node) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$n'. (undefined)

(UnusedFormalParameter)